### PR TITLE
3.x branch: ARM OSX Migrator (again)

### PR DESCRIPTION
**Pull request by stuarteberg**
This is my own copy of #25 to play with, since I can't push to that PR.

Closes https://github.com/conda-forge/libsigcpp-feedstock/pull/25

### CODE REVIEW

**Comment by stuarteberg** [comment]
I "fixed" the 3.x build here by simply skipping the `make check` step during cross-compilation (as suggested in the [conda-forge blog][1]).  Is that okay?

The weird thing is that no such skipping was required on the [2.x branch][2].  That makes me wonder if something is actually going wrong here.  How can I determine if `make check` ought to have passed here, just as `ninja test` passed in the 2.x branch?

[1]: https://conda-forge.org/blog/posts/2020-10-29-macos-arm64/#how-to-add-a-osx-arm64-build-to-a-feedstock
[2]: https://github.com/conda-forge/libsigcpp-feedstock/pull/27/files#diff-eacd3114061551f6510696ecb6b9ac03fffbf2f31c699750fe1ea706a764065bR17

File: conda-forge.yml [hunk 1]
```diff
@@ -1,2 +1,7 @@
+build_platform:
```
Comment (jakirkham) [inline conda-forge.yml:4]: Would suggest adding this to make sure the `2.x` branch is getting other migration PRs

```suggestion
bot:
  abi_migration_branches:
  - 2.x
build_platform:
```

**Comment by jakirkham** [comment]
@conda-forge/libsigcpp, could you please review this and PR ( https://github.com/conda-forge/libsigcpp-feedstock/pull/27 )?

**Comment by stuarteberg** [comment]
>@conda-forge/libsigcpp, could you please review this and PR ( #27 )?

If time is scarce, please prioritize #27 (not this one).  As far as I can tell, [no downstream conda-forge packages use the 3.x ABI yet](https://github.com/search?q=org%3Aconda-forge+sigcpp&type=code), so this one is less important.

**Pull request merged**